modules/microshift-cni-customization-matrix.adoc

```diff
@@ -0,0 +1,46 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-nw-ovn-kubernetes-matrix_{context}"]
+= {microshift-short} CNI feature matrix
+
+The following table summarizes the status of OVN-K networking features that are either present as defaults, supported for configuration, or excluded from the {microshift-short} service:
+
+.Default {microshift-short} OVN-K network plugin features and customization status
+[cols="50%,20%,30%",options="header"]
+|===
+|Network feature|Availability|Customization status
+
+|Kubernetes network policy|Yes|Supported
+
+|Kubernetes network policy logs|Not available|N/A
+
+|Load balancing|Yes|Supported
```

Comment:
This looks unclear to me. Does it mean the support to the LoadBalancer type of k8s Services?